code/game/objects/items/RCD.dm

```diff
@@ -32,11 +34,13 @@ ARCD
 
 /obj/item/construction/Initialize()
 	. = ..()
-	desc = "\A [src]. It currently holds [matter]/[max_matter] matter-units."
 	spark_system = new /datum/effect_system/spark_spread
 	spark_system.set_up(5, 0, src)
 	spark_system.attach(src)
 
+/obj/item/construction/examine(mob/user)
```

Comment:
RCD was literally setting desc in init/new to state this below. So I just changed it to one location. In a examine